mmv1/third_party/terraform/services/vertexai/resource_vertex_ai_rag_engine_config_test.go

```diff
@@ -0,0 +1,37 @@
+package vertexai_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+func TestAccVertexAIRagEngineConfig_basic(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"project": envvar.GetTestProjectFromEnv(),
+	}
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		Steps: []resource.TestStep{
```

Comment:
Everything is otherwise LGTM, but could you add some update steps? In particular, it'd be good to see an update step that goes to `unprovisioned` so that the custom delete is getting exercised regularly.